openlibrary/catalog/marc/tests/test_data/bin_expect/830_series.json

```diff
@@ -0,0 +1,42 @@
+{
+  "publish_date": "1969",
+  "publish_country": "ja",
+  "languages": [
+    "eng"
+  ],
+  "title": "Capital versus labour under high growth",
+  "lccn": [
+    "75577579"
+  ],
+  "authors": [
+    {
+      "birth_date": "1921",
+      "name": "Kimizuka, Yoshiro",
+      "entity_type": "person",
+      "personal_name": "Kimizuka, Yoshiro"
+    }
+  ],
+  "lc_classifications": [
+    "HD2907 .K55"
+  ],
+  "series": [
+    "The Science Council of Japan. Division of Economics, Commerce & Business Administration. Economic series no. 46",
+    "Economic series (Nihon Gakujutsu Kaigi. Dai 3-bu) -- no. 46"
+  ],
```

Comment:
I don't think it's important given OpenLibrary's current liberal cataloging practices, but there's an extensive discussion of MARC 490 vs 830 at Yale https://web.library.yale.edu/cataloging/CIP/editing-490-830 It may make sense at some point to drop the 490 1 and only use the corresponding 8xx for series which are "traced". Of course 490 0 records would always be imported as is.

Of course the nice thing about the 490 is that it's what appears on the volume, so I can see a place for both.